app/assets/stylesheets/govuk_publishing_components/components/_big-number.scss

```diff
@@ -58,3 +58,27 @@
 .gem-c-big-number__suffix {
   vertical-align: middle;
 }
+
+@include govuk-media-query($media-type: print) {
+  .gem-c-big-number__value {
+    font-size: 56pt;
+  }
+
+  .gem-c-big-number__suffix {
+    vertical-align: unset;
```

Comment:
Nitpick: is this needed? Oddly in screen when this is removed the plus moves up, but in print it moves down - but either way I don't think it looks bad.

Reply:
This component actually needs more attention (possible refactor) because the screen styles for the suffix are broken: they don't work with different types of suffix, for example % or letters. However I didn't want to refactor the screen styles as part of this work which is print focussed. (I've recorded this as something to look at in the future).

The `vertical-align: unset` fixes the incorrect vertical centring of other characters, which you can see if you use dev tools to change the + in the example to %, £, M etc. Everything now lines up much better. 

That said, perhaps it's strange to fix this for print and not screen, so I'd be happy to remove it here and defer the whole thing to a different ticket to fix the entire component if that makes more sense?

Reply:
Yes, I think that makes more sense. Can you raise an issue in the repo detailing this? Thanks.

Reply:
I've deleted this commit and raised an issue here: [4159](https://github.com/alphagov/govuk_publishing_components/issues/4159)